python/activator/middleware_interface.py

```diff
@@ -610,6 +610,8 @@ def _export_skymap_and_templates(self, center, detector, wcs, filter):
                 collections=self._collection_template,
                 instrument=self.instrument.getName(),
                 skymap=self.skymap_name,
+                tract=tract.tract_id,
+                physical_filter=filter,
```

Comment:
For my own curiousity, do you see this going meaningfully faster in a single query (or a small number of queries), or is it only when doing many queries that it becomes noticeable?

Reply:
Not "meaningfully", no -- templates went from 0.0801, 0.0834 to 0.0697, 0.0678, 0.0840, 0.0846, 0.0721. But if nothing else, it will keep the code from getting cargo-culted to cases where it _does_ matter.